svg-spritemap.js

```diff
@@ -80,7 +80,7 @@ SVGSpritemapPlugin.prototype.apply = function(compiler) {
                 var SVGOptimizer = new svgo(options.svgo);
                 var filename = chunk.files[1];
 
-                SVGOptimizer.optimize(compilation.assets[filename].source(), function(o) {
+                SVGOptimizer.optimize(compilation.assets[filename].source()).then(function(o) {
```

Comment:
I'd rather keep working with callbacks in this version. I'm working on a `webpack@^4.0.0` version of the plugin which will be refactored to ES6.